API/testing/app.js

```diff
@@ -0,0 +1,115 @@
+const dotenv = require('dotenv');
+// Environment Variable Import
+dotenv.config({ path: '.env' });
+
+const AWS = require("aws-sdk");
+
+const uuid = require('uuid');
+
+// HTTP Response
+let response;
+let gamelift = new AWS.GameLift();
+
+/**
+ *
+ * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @param {Object} event - API Gateway Lambda Proxy Input Format
+ *
+ * Context doc: https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html 
+ * @param {Object} context
+ *
+ * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+ * @returns {Object} object - API Gateway Lambda Proxy Output Format
+ * 
+ */
+exports.lambdaHandler = async (event, context) => {
+    try {
+        console.log(event);
+        switch (event.resource) {
+            // Test Game Session POST - Create Game Session
+            case '/test/game/session':
+                console.log('Getting GameLift Session');
+                let params = {
+                    MaximumPlayerSessionCount: '2', /* required */
+                    //AliasId: 'STRING_VALUE',
+                    //CreatorId: 'STRING_VALUE',
+                    FleetId: process.env.FLEETID,
+                    GameProperties: [
+                        {
+                            Key: 'test', /* required */
+                            Value: 'STRING_VALUE' /* required */
+                        },
+                        /* more items */
```

Comment:
What items do you need to add here?

Reply:
Those are in case we need to add more game properties.